.changes/config.json

```diff
@@ -0,0 +1,171 @@
+{
+  "gitSiteUrl": "https://github.com/tauri-apps/plugins-workspace/",
+  "pkgManagers": {
+    "javascript": {
+      "version": true,
+      "getPublishedVersion": "pnpm view ${ pkgFile.pkg.name } version",
+      "publish": ["pnpm build", "pnpm publish --access public --no-git-checks"]
+    },
+    "rust": {
+      "version": true,
+      "getPublishedVersion": "cargo search ${ pkgFile.pkg.package.name } --limit 1 | sed -nE 's/^[^\"]*\"//; s/\".*//1p' -",
+      "publish": [
+        {
+          "command": "cargo package --no-verify",
+          "dryRunCommand": true
+        },
+        {
+          "command": "echo '<details>\n<summary><em><h4>Cargo Publish</h4></em></summary>\n\n```'",
+          "dryRunCommand": true,
+          "pipe": true
+        },
+        {
+          "command": "cargo publish",
+          "dryRunCommand": "cargo publish --dry-run",
+          "pipe": true
+        },
+        {
+          "command": "echo '```\n\n</details>\n'",
+          "dryRunCommand": true,
+          "pipe": true
+        }
+      ]
+    }
+  },
+  "packages": {
+    "authenticator": {
+      "path": "./plugins/authenticator",
+      "manager": ""
```

Comment:
why aren't you specifying a manager?

Reply:
so that covector skips it since we don't want to publish it yet

Reply:
would be prettier if json would support comments :D (or if covector would have an explicit off switch maybe? idk)

Reply:
ah, well, everything looks fine to me